README.md

```diff
@@ -61,9 +61,15 @@ After you're finished please remove all the comments and instructions!
 
 ## 🛠 Built With <a name="built-with"></a>
 
-. This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- React
+- JavaScript
+- CSS
+- ReactJS
+- GitHub-Pages
+- Eslint
+- Stylelint
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+(<a href="#readme-top">back to top</a>)
 
 <!-- LIVE DEMO -->
 
```

Comment:
- [ ] OPTIONAL
Live demo link in your ReadMe is broken, kindly endeavor to fix it.